2011-01-21  Nicolas Setton  <setton@adacore.com>

	* configure: Add missing file to the list of config_files.

*** configure.original	2011-01-21 14:52:41.000000000 +0100
--- configure	2011-01-21 14:52:54.000000000 +0100
***************
*** 14597,14603 ****
  
  
  
! ac_config_files="$ac_config_files Makefile data/gladeui-1.0.pc data/glade-3.desktop.in data/Makefile data/icons/Makefile data/icons/hicolor/Makefile data/icons/hicolor/16x16/Makefile data/icons/hicolor/16x16/apps/Makefile data/icons/hicolor/22x22/Makefile data/icons/hicolor/22x22/apps/Makefile data/icons/hicolor/24x24/Makefile data/icons/hicolor/24x24/apps/Makefile data/icons/hicolor/32x32/Makefile data/icons/hicolor/32x32/apps/Makefile data/icons/hicolor/48x48/Makefile data/icons/hicolor/48x48/apps/Makefile data/icons/hicolor/scalable/Makefile data/icons/hicolor/scalable/apps/Makefile gladeui/Makefile gladeui/gladeui.rc src/Makefile src/glade-3.rc plugins/Makefile plugins/gtk+/Makefile plugins/gtk+/icons/Makefile plugins/gtk+/icons/16x16/Makefile plugins/gtk+/icons/22x22/Makefile plugins/gnome/Makefile plugins/gnome/icons/Makefile plugins/gnome/icons/16x16/Makefile plugins/gnome/icons/22x22/Makefile plugins/python/Makefile po/Makefile.in doc/Makefile doc/version.xml help/Makefile"
  
  
  cat >confcache <<\_ACEOF
--- 14597,14603 ----
  
  
  
! ac_config_files="$ac_config_files Makefile data/gladeui-1.0.pc data/glade-3.desktop.in data/Makefile data/icons/Makefile data/icons/hicolor/Makefile data/icons/hicolor/16x16/Makefile data/icons/hicolor/16x16/apps/Makefile data/icons/hicolor/22x22/Makefile data/icons/hicolor/22x22/apps/Makefile data/icons/hicolor/24x24/Makefile data/icons/hicolor/24x24/apps/Makefile data/icons/hicolor/32x32/Makefile data/icons/hicolor/32x32/apps/Makefile data/icons/hicolor/48x48/Makefile data/icons/hicolor/48x48/apps/Makefile data/icons/hicolor/scalable/Makefile data/icons/hicolor/scalable/apps/Makefile gladeui/Makefile gladeui/gladeui.rc src/Makefile src/glade-3.rc plugins/Makefile plugins/gtk+/Makefile plugins/gtk+/icons/Makefile plugins/gtk+/icons/16x16/Makefile plugins/gtk+/icons/22x22/Makefile plugins/gnome/Makefile plugins/gnome/icons/Makefile plugins/gnome/icons/16x16/Makefile plugins/gnome/icons/22x22/Makefile plugins/python/Makefile po/Makefile.in doc/Makefile doc/version.xml help/Makefile gladeui/glade-previewer.rc"
  
  
  cat >confcache <<\_ACEOF
